app/models/configuration_script.rb

```diff
@@ -1,11 +1,4 @@
-class ConfigurationScript < ApplicationRecord
-  serialize :variables
-  serialize :survey_spec
-
-  acts_as_miq_taggable
-
-  belongs_to :inventory_root_group, :class_name => "EmsFolder"
-  belongs_to :manager,              :class_name => "ExtManagementSystem"
-
-  include ProviderObjectMixin
+class ConfigurationScript < ConfigurationScriptBase
+  # As a job template, I have a playbook
+  belongs_to :configuration_script_payload
```

Comment:
I feel like this belongs on the base class implemented as
```ruby
belongs_to :parent
has_many :children, :class => "ConfigurationScriptBase", :foreign_key => "parent_id"
```
What do you think @Fryguy ?